src/Authentication/Authentication/Cmdlets/SetMgGraphOption.cs

```diff
@@ -0,0 +1,41 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using System.Management.Automation;
+
+namespace Microsoft.Graph.PowerShell.Authentication.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Set, "MgGraphOption", HelpUri = "")]
+    public class SetMgGraphOption : PSCmdlet
+    {
+        protected override void BeginProcessing()
+        {
+            base.BeginProcessing();
+        }
+
+        protected override void ProcessRecord()
+        {
+            base.ProcessRecord();
+            GraphSession.Instance.GraphOption.EnableWAMForMSGraph = !GraphSession.Instance.GraphOption.EnableWAMForMSGraph;
+            if (GraphSession.Instance.GraphOption.EnableWAMForMSGraph == true)
+            {
```

Comment:
Please add a Boolean or switch parameter that customers can use to toggle login by WAM. Customers should be able to use the command as such: `Update-MgGraphOption -EnableLoginByWAM $true` to enable WAM and `Update-MgGraphOption -EnableLoginByWAM $false` to disable login by WAM.
```suggestion
            if (EnableLoginByWAM)
            {
                GraphSession.Instance.GraphOption.EnableWAMForMSGraph = EnableLoginByWAM;
```